client/src/PublicRequest.ts

```diff
@@ -78,6 +78,7 @@ export type SimpleRequest = BasicRequest & {
 export type ExportRequest = SimpleRequest & {
     fileOnly?: boolean,
     wordsOnly?: boolean,
+    useLegacyStore?: boolean,
```

Comment:
in case automatically detecting from which store to read this would be unnecessary.